src/js/components/returns/outbound/SendOutboundReturn.jsx

```diff
@@ -481,7 +481,7 @@ class SendMovementPage extends Component {
                   disabled={values && values.status === 'COMPLETED'}
                   onClick={() => this.previousPage(values, invalid)}
                 >
-                  <Translate id="react.default.button.previous.label" defaultMessage="Previous" />
```

Comment:
this one should be working

Reply:
yeah, but it's not consistent with the rest pages from this workflow